### ISSUE TITLE
osx GUI disappeared

### ISSUE THREAD

**Issue by exiledsurfer** [keybase/client#8556]
Keybase GUI Version: 1.0.30-20170915193210+8c3c799f2

closed a chat channel in the osx gui client, and now the window is only white, and no menu items work. 

just reinstall?

**Comment by exiledsurfer** [comment]
my log id: 6c5614f4888ec1dab6b0cb1c

**Comment by exiledsurfer** [comment]
additionally: keybase logs do not appear in console sidebar
![screen shot 2017-09-20 at 11 55 29](https://user-images.githubusercontent.com/1099799/30638125-ae7e8f4e-9dfa-11e7-90a5-3f2835cf307d.png)

**Comment by exiledsurfer** [comment]
i see this was reported here https://github.com/keybase/client/issues/8189 , but no resolution?

**Comment by maxtaco** [comment]
We know what the issue is but can't repro it.  MacOS's launchd just
pretends to accept our commands but throws them on the floor. We continue
to scratch our heads in frustration.

Does the problem resolve itself after a reboot of your machine?

**Comment by Heliosmaster** [comment]
@maxtaco for me, after killing and restarting Keybase, worked fine. A few months ago (#6965) that wouldn't work. So at least there's some mitigation now, hopefully.

**Comment by exiledsurfer** [comment]
@maxtaco well, that seems to have done the trick... old school style when everything else fails reboot (kicks self) 
@Heliosmaster & max: restart of the app itself didn't work for me and i didn't want to reinstall w/out an answer to the ticket first.

so the issue is closed for my current needs :-)

**Comment by maxtaco** [comment]
Thanks for the feedback.  I've seen the app in this state just once, and a restart fixed it.  But we've been otherwise able to repro the issue.

**Comment by Heliosmaster** [comment]
@exiledsurfer yeah reboot of the machine always worked for me as well. It seems that this weird situation is triggered after a sleep

**Comment by chrisnojima** [comment]
after it works, you can get it back into this state (through a sleep?)
